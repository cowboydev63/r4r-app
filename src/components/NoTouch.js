import React from 'react';
import { Theme } from '../theme';

const NoTouch = props => (
    <Theme
        element="div"
        className="notouch-overlay"
    >
        { props.children }
    </Theme>
)

export default NoTouch;