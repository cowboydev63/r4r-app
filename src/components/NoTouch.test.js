import React from 'react';
import { shallow } from 'enzyme';
import NoTouch from './NoTouch';

describe('NoTouch Component', () => {
  it('renders without crashing', () => {
    shallow(<NoTouch />);
  });

  it('renders correctly', () => {
    const wrapper = shallow(<NoTouch/>);
    expect(wrapper).toMatchSnapshot();
  })
})