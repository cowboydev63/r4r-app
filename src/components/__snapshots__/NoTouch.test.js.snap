// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoTouch Component renders correctly 1`] = `
<Theme
  className="notouch-overlay"
  element="div"
/>
`;
